### PR TITLE
Change azure pipeline

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: src/typescript/azure-openapi-validator/rule.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ interface ValidationMessage {
 }
 
 export interface Rule {
+
```
Comment (ruowan) [inline src/typescript/azure-openapi-validator/rule.ts]: Why the empty line here?

File: azure-pipelines/ci.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+trigger:
```
Comment (ruowan) [inline azure-pipelines/ci.yml]: you can directly use azure-pipelines.yml in root folder level instead of putting it into a folder.

Comment (qiaozha) [reply azure-pipelines/ci.yml]: to test pipeline

Comment (qiaozha) [reply azure-pipelines/ci.yml]: I want to create a release pipeline later to separate it

File: src/typescript/azure-openapi-validator/rule.ts (hunk 1 above)
Comment (qiaozha) [reply src/typescript/azure-openapi-validator/rule.ts]: to test pipeline

File: azure-pipelines/ci.yml (hunk 2 above)
Comment (ruowan) [reply azure-pipelines/ci.yml]: For release pipeline. It's private you can't put it into here.

Comment (qiaozha) [reply azure-pipelines/ci.yml]: I mean in a separate file but in azure-pipelines folder. That's why I create a folder. And the release pipeline will sure be put in internal pipelines.

File: src/typescript/azure-openapi-validator/rule.ts (hunk 1 above)
Comment (ruowan) [reply src/typescript/azure-openapi-validator/rule.ts]: For testing pipeline. You can comments /azp run.  So you can revert the changes, it's not necessary.

File: azure-pipelines/ci.yml (hunk 2 above)
Comment (ruowan) [reply azure-pipelines/ci.yml]: Release pipeline YAML  file can't be put into GitHub repo.   It is stored in DevOps. So this repo only contains one pipeline. Create a folder will make it no clear.

File: src/typescript/azure-openapi-validator/rule.ts (hunk 1 above)
Comment (qiaozha) [reply src/typescript/azure-openapi-validator/rule.ts]: changed

File: azure-pipelines/ci.yml (hunk 2 above)
Comment (qiaozha) [reply azure-pipelines/ci.yml]: changed

**Pull request merged**